docs/dev-guide.md

```diff
@@ -906,27 +906,6 @@ To automatically fix lint and formatting errors, run `make format`.
 
 - There are three levels at which “open” status is tracked, as follows. If `open = false` for any object then it will block the creation of new objects below it. For example, to create a new submission the corresponding variant, instance_question, and assessment_instance must all be open.
 
-| Variable                   | Allow new `instance_questions` | Allow new `variants` | Allow new `submissions` |
-| -------------------------- | ------------------------------ | -------------------- | ----------------------- |
-| `assessment_instance.open` | ✓                              | ✓                    | ✓                       |
-| `instance_question.open`   |                                | ✓                    | ✓                       |
-| `variant.open`             |                                |                      | ✓                       |
```

Comment:
I don't know why I removed this - this looks like it's still useful. Maybe a merge conflict? I'll put it back.

Reply:
Resolved in 193e629.